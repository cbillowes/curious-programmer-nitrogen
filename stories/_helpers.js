import React from 'react'
import Layout from '../src/components/layout'
import { SmallSpacer } from '../src/components/spacer'

const Page = ({ className, children }) => {
  return (
    <Layout
      className={className}
    >
      <SmallSpacer />
      {children}
    </Layout>
  )
}

export default Page