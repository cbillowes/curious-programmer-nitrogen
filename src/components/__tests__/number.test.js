import React from 'react'
import Number from '../number'
import { getSnapshot } from './_helpers'

describe(`Navigation`, () => {
  it(`should render empty when there is no number`, () => {
    const component = (
      <Number />
    )
    const tree = getSnapshot(component)
    expect(tree).toMatchSnapshot()
  })

  it(`should render`, () => {
    const component = (
      <Number
        number={42}
      />
    )
    const tree = getSnapshot(component)
    expect(tree).toMatchSnapshot()
  })
})
