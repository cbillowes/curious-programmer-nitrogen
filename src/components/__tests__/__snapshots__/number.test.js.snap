// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation should render 1`] = `
<div
  className="post number ribbon"
>
  <div
    className="backdrop"
  >
    <div
      className="text"
    >
      #
      42
    </div>
  </div>
</div>
`;

exports[`Navigation should render empty when there is no number 1`] = `null`;
